renovate/defaults.json5

```diff
@@ -198,5 +198,12 @@ Validate this file before commiting with (from repository root):
       // Don't wait, roll out CI VM Updates immediately
       "schedule": ["at any time"],
     },
+
+    // Add CI:DOCS prefix to skip unnecessary tests for golangci updates in podman CI.
+    {
+      "matchManagers": ["regex"],
```

Comment:
```suggestion
```

I think this might be a situation where "less is more".  It would be desirable (I think) for this to match *both* the regex updates and the regular package updates.  I think having just the `matchPackagenames` should do that.

The docs are confusing on this, since there is also a `matchDepNames`.  But [the docs say the regex manager will set the packageName based on the depName by default](https://docs.renovatebot.com/configuration-options/#packagenametemplate).  So I think this is the right thing to do.